.travis.yml

```diff
@@ -17,16 +15,10 @@ script:
   - npm run build:rollup
   - npm --prefix=server test
 
-before_install:
-  - npm install -g npm@6.9.0
-  - node --version
-  - npm --version
-
 cache: npm
 
 install:
-  - npm ci
-  - npm --prefix=server ci
+  - ./tools/setup
```

Comment:
it seems that travis supports doing the `npm ci` step by itself?

It also appears to support caching of npm modules too -- perhaps follow up on that at some point?

https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#dependency-management

Reply:
I think travis can do all of the setup without the use of the script. However, we want to ensure that the setup script is not brittle (or we know when it goes stale). Thus, we want to exercise it.